workshops/templates/base.html

```diff
@@ -64,17 +68,29 @@
           </form>
 
           <ul class="nav navbar-nav navbar-right">
+            {% if perms.workshops.add_event or perms.workshops.add_host or perms.workshops.add_person or perms.workshops.add_airport %}
```

Comment:
With "gray-out" we don't need to check this long chain of permissions. We'll only check individual URL.